relayer/relayer/src/main.rs

```diff
@@ -274,6 +269,25 @@ async fn run_relayer(
         create_eth_connections(&config, persistent_eth_connection).await?;
     info!("Established connection to the Ethereum node");
 
+    let most_azero = MostInstance::new(
+        &config.azero_contract_address,
+        &config.azero_contract_metadata,
+        config.azero_ref_time_limit,
+        config.azero_proof_size_limit,
+    )?;
+
+    let current_committee_id = most_azero
+        .current_committee_id(azero_connection.as_connection())
+        .await?;
+    most_azero
+        .set_payout_account(
+            &azero_signed_connection,
+            current_committee_id,
+            AccountId::from_string(&config.payout_address)
+                .map_err(|why| AzeroContractError::NotAccountId(why.to_string()))?,
```

Comment:
This should be backward compatible - let's return `None` if there's no  such entry `config`

Reply:
I have changed this such that the parameter is now optional.